tools/mkdeps.c

```diff
@@ -81,12 +81,12 @@
  * Private Types
  ****************************************************************************/
 
-enum slashmode_e
+typedef enum
 {
-  MODE_FSLASH  = 0,
-  MODE_BSLASH  = 1,
-  MODE_DBLBACK = 2
-};
+  TOOLCHAIN_GNU      = 0,
```

Comment:
should we use the same approach as incdir.c for compiler detecting?

Reply:
emm... consider whether it is inappropriate to decide on behavior during compilation? If the toolchain is changed after first configure